### DISCUSSION TITLE
Can this be used to automate refresh of Cloud Managed (VM) from Microsoft hosted (Tier 1)?

### DISCUSSION THREAD

**Discussion by MarkGTI** [d365collaborative/d365fo.tools#602] [Q&A]
This is what we have to do manually right now:

No. | Task | Time in Min
1   | Sandbox backup - Export DB | 30
2   | VM DB backup - Manual DB backup from SSMS | 15
3   | PROD to Sandbox refresh from LCS | 60
4   | Sandbox backup - Export DB ( After prod refresh) | 30
5   | Copying bacpack file to destination server | 15
6   | Preparing command to convert bacpack file to bak | 20
7   | Running command to convert bacpack file to bak | 140
8   | Import Database to Sandbox (previously taken at step 1) | 120
9   | Stopping application service | 20
10 | Running preparation script | 15
11 | Backup of Sandbox COPY database after step 10 | 15
12 | Restore step 11 backup to AX DB | 30
13 | Running update database script on AXDB | 15
14 | Starting application service | 20
15 | Verifying Sandbox env | 30
16 | Verifying non prod env | 30

Trying to figure out how to automate

**Comment by Splaxi** [comment]
I would argue that 1-7 is not a problem. 

8 is supported from an automated perspective 

9-14 not a problem. 

The last 2 isn't all to clear for me. 

What would you like to know more about?

**Comment by MarkGTI** [comment]
Thanks for reply

Right now, just trying to figure out if it's do-able and will give to our new DevOps guy to see if he can figure it out. Didn't want to send him down a rabbit hole

From: Mötz Jensen ***@***.***>
Sent: Wednesday, November 10, 2021 4:19 PM
To: d365collaborative/d365fo.tools ***@***.***>
Cc: Mark Anderson ***@***.***>; Author ***@***.***>
Subject: [EXTERNAL] Re: [d365collaborative/d365fo.tools] Can this be used to automate refresh of Cloud Managed (VM) from Microsoft hosted (Tier 1)? (Discussion #602)


I would argue that 1-7 is not a problem.

8 is supported from an automated perspective

9-14 not a problem.

The last 2 isn't all to clear for me.

What would you like to know more about?

-
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://nam11.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fd365collaborative%2Fd365fo.tools%2Fdiscussions%2F602%23discussioncomment-1622673&data=04%7C01%7C%7C66b2cda371d14f84130608d9a498115a%7C080c6c8ebced4e09b5c36e55cbd80da4%7C0%7C0%7C637721795310164479%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=GshjP204XekvbkzdzWFefmi13ndvYSRjOmLmv9TIL6U%3D&reserved=0>, or unsubscribe<https://nam11.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAWNWAHAWHTW3NCIVKBGWYVDULLVUPANCNFSM5HYLXIVA&data=04%7C01%7C%7C66b2cda371d14f84130608d9a498115a%7C080c6c8ebced4e09b5c36e55cbd80da4%7C0%7C0%7C637721795310164479%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=HUePtrs9z6IDOtC8t%2BZxIT2DL8xfje%2FP5Lzj8kowfnc%3D&reserved=0>.
Triage notifications on the go with GitHub Mobile for iOS<https://nam11.safelinks.protection.outlook.com/?url=https%3A%2F%2Fapps.apple.com%2Fapp%2Fapple-store%2Fid1477376905%3Fct%3Dnotification-email%26mt%3D8%26pt%3D524675&data=04%7C01%7C%7C66b2cda371d14f84130608d9a498115a%7C080c6c8ebced4e09b5c36e55cbd80da4%7C0%7C0%7C637721795310174474%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=QzboCos2hKqRb%2BFqv9fP4%2BnGSABT0v43Jcx4RoryDEU%3D&reserved=0> or Android<https://nam11.safelinks.protection.outlook.com/?url=https%3A%2F%2Fplay.google.com%2Fstore%2Fapps%2Fdetails%3Fid%3Dcom.github.android%26referrer%3Dutm_campaign%253Dnotification-email%2526utm_medium%253Demail%2526utm_source%253Dgithub&data=04%7C01%7C%7C66b2cda371d14f84130608d9a498115a%7C080c6c8ebced4e09b5c36e55cbd80da4%7C0%7C0%7C637721795310174474%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=ot3ohDyIztwqmNpWVu6cFxiAp9VJNuuwZdjRWHssAao%3D&reserved=0>.

**Comment by Splaxi** [comment]
Sure thing.

I have to correct my answer for no. 8. It is NOT possible at the moment. But it could be handle in many different ways or you could ask Microsoft if they would consider supporting it.

**Comment by MarkGTI** [comment]
Thanks

From: Mötz Jensen ***@***.***>
Sent: Thursday, November 11, 2021 12:38 AM
To: d365collaborative/d365fo.tools ***@***.***>
Cc: Mark Anderson ***@***.***>; Author ***@***.***>
Subject: [EXTERNAL] Re: [d365collaborative/d365fo.tools] Can this be used to automate refresh of Cloud Managed (VM) from Microsoft hosted (Tier 1)? (Discussion #602)


Sure thing.

I have to correct my answer for no. 8. It is NOT possible at the moment. But it could be handle in many different ways or you could ask Microsoft if they would consider supporting it.

-
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://nam11.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fd365collaborative%2Fd365fo.tools%2Fdiscussions%2F602%23discussioncomment-1624001&data=04%7C01%7C%7C0750e00a481d42b0e67808d9a4dddbb5%7C080c6c8ebced4e09b5c36e55cbd80da4%7C0%7C0%7C637722095071280693%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=A9SReNKmXDP4VFhEAGOews%2BVe8BE%2FBu3udnqeGkZyDI%3D&reserved=0>, or unsubscribe<https://nam11.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAWNWAHCMKXK5PI6JRIYDIXLULNQF3ANCNFSM5HYLXIVA&data=04%7C01%7C%7C0750e00a481d42b0e67808d9a4dddbb5%7C080c6c8ebced4e09b5c36e55cbd80da4%7C0%7C0%7C637722095071290688%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=k5dmei9ofYssyCBZjjS5qH3WX78UrKkNW18b6aXBsDI%3D&reserved=0>.
Triage notifications on the go with GitHub Mobile for iOS<https://nam11.safelinks.protection.outlook.com/?url=https%3A%2F%2Fapps.apple.com%2Fapp%2Fapple-store%2Fid1477376905%3Fct%3Dnotification-email%26mt%3D8%26pt%3D524675&data=04%7C01%7C%7C0750e00a481d42b0e67808d9a4dddbb5%7C080c6c8ebced4e09b5c36e55cbd80da4%7C0%7C0%7C637722095071300683%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=Ofp%2FF2dUMP6SrQ%2FP5nfc37Agfro2tFfLkaksBenjDoY%3D&reserved=0> or Android<https://nam11.safelinks.protection.outlook.com/?url=https%3A%2F%2Fplay.google.com%2Fstore%2Fapps%2Fdetails%3Fid%3Dcom.github.android%26referrer%3Dutm_campaign%253Dnotification-email%2526utm_medium%253Demail%2526utm_source%253Dgithub&data=04%7C01%7C%7C0750e00a481d42b0e67808d9a4dddbb5%7C080c6c8ebced4e09b5c36e55cbd80da4%7C0%7C0%7C637722095071300683%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=K7Br6Ok0VPNJiHa3H%2BHQ2rxp9Kaheilv3t90zsSCLoc%3D&reserved=0>.
